stdlib/tkinter/__init__.pyi

```diff
@@ -277,11 +279,11 @@ def NoDefaultRoot() -> None: ...
 _TraceMode: TypeAlias = Literal["array", "read", "write", "unset"]
 
 class Variable:
-    def __init__(self, master: Misc | None = ..., value: Any | None = ..., name: str | None = ...) -> None: ...
-    def set(self, value: Any) -> None: ...
+    def __init__(self, master: Misc | None = ..., value: Incomplete | None = ..., name: str | None = ...) -> None: ...
```

Comment:
Cases like this should just be kept unannotated. `Incomplete` is intended for partially unknown types, which we can't just leave unannotated.

Reply:
I was going to say that `Any | None` is more type-safe than `Any`, but I guess it doesn't actually give you any more type safety for argument types, only for attribute annotations and return types.

Reply:
It still makes sense to use `Incomplete | None` in argument positions as it's one more step towards correctness and at least serves a documentarion.

Reply:
> It still makes sense to use `Incomplete | None` in argument positions as it's one more step towards correctness and at least serves a documentarion.

Oh, I thought that was what you were arguing against here. Which specific change are you commenting on? :)

Reply:
Oops, seems I misclicked the line. That happens when doing stuff like this on the phone. I mean cases like `set` and `get` below.

Reply:
Ignore everything I said in response to your comment in that case; we're in agreement 😄

Reply:
Indeed. I deleted `Incomplete`s like this manually, but apparently missed a couple.